app/controllers/api/v1/participants_controller.rb

```diff
@@ -2,8 +2,40 @@ module Api::V1
   class ParticipantsController < ApplicationController
 
     def index
-      render json: Participant.all
+      render json: Participant.where(status: 'accepted')
     end
 
+    def create
+      participant = Participant.create(email: params[:email],
```

Comment:
@mdonagh I see you are specifying the params we want... what keeps us safe..any way I think we could use the [strong parameters conversion](https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters) of creating a private method where we can specify permitted params. I mean this:

```
private
    # Using a private method to encapsulate the permissible parameters
    # is just a good pattern since you'll be able to reuse the same
    # permit list between create and update. Also, you can specialize
    # this method with per-user checking of permissible attributes.
    def person_params
      params.require(:person).permit(:name, :age)
    end

```